src/docs/product/session-replay/protecting-user-privacy.mdx

```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
 
 To learn more, including available configuration options, see [Session Replay Privacy](/platforms/javascript/session-replay/privacy/).
 
-## User Input
+### User Input
 
 In addition to HTML content, Session Replay records key user interactions like mouse movements, mouse clicks, and keyboard presses. By default, all user keypresses are redacted and replaced with asterisks. Captured mouse movements are limited to source and target destinations (for example, where the mouse came from and where it went, not the exact path). A low-fidelity approximation of this movement is recreated during playback.
```

Comment:
>... like mouse movements, mouse clicks, and keyboard presses.

Should we clarify that these are keypresses into text-capture areas, ie: inputs with focus? Feels like a technicality but might spark an idea

Reply:
Yes we should, I will follow up.